rel/files/mongooseimctl

```diff
@@ -219,8 +239,8 @@ done
 
 case $1 in
     'start') start;;
-    'add_to_cluster') add_to_cluster;;
-    'remove_from_cluster') remove_from_cluster;;
+    'join_cluster') join_cluster;;
+    'leave_cluster') leave_cluster;;
```

Comment:
As far as I understand leave_cluster should be run remove a node (different one) from a cluster. If we have 3 nodes A, B, C and we want to remove C from the cluster. Then we need to run this command on A or B. Am I right? If so the `leave_cluster` name is not intuitive to me.